tests/utils/database.test.ts

```diff
@@ -0,0 +1,159 @@
+import { expect } from 'chai';
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+import { Database } from '../../src/utils/database';
+
+const database = new Database();
+
+describe('database', () => {
+  context('hasGameFile', () => {
```

Comment:
Change to describe per our [testing practices](https://github.com/normal-devs/klicker-knight/issues/7)

Reply:
Done!